hoot-core/src/main/cpp/hoot/core/io/ServicesDbReader.cpp

```diff
@@ -94,7 +94,39 @@ void ServicesDbReader::_addTagsToElement(shared_ptr<Element> element)
     element->setCircularError(tags.get("error:circular").toDouble(&ok));
     if (!ok)
     {
-      LOG_WARN("Error parsing error:circular.");
+      //check if value contian velocity units
+      if (tags.get("error:circular").contains("km/h", Qt::CaseInsensitive)
+          || tags.get("error:circular").contains("kph", Qt::CaseInsensitive)
+          || tags.get("error:circular").contains("kmph", Qt::CaseInsensitive)
+          || tags.get("error:circular").contains("mph", Qt::CaseInsensitive)
+          || tags.get("error:circular").contains("knots", Qt::CaseInsensitive))
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.